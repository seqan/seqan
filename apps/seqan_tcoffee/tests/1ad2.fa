>rk1_cyapa
EEQLYEPLEAIQLLKQISTAKFIETVEVHCRLGIDPKYNDQQLRATVTKAGKLEFRNDRTGIVHVLFGKA
NFSEEALLGNLKAVQEAVDRNRPSGVKGKYWKSIYITSTMSPSIEVDLPNGTGQTLRVAVITRGEKINEA
STAGADLVGAEELIDEIQNGRLDFDCLIATPDMMPQVAKLGRVLGPRGLMPSPKGGTVTFDLAQAINDF
>rl1_halvo
VDAVSRALDEAPGRNFRETVDLAVNLRDLDLNDPSKRVDESIVADETDFFVAEAGLMQDIGRYLGTVLGP
RGKMPTPLQPDDDVVETVNRMKNTVQLRSRDRRTFHTRVGADDMTPDEIAENIDVIVRRLEATLEKGPLN
IDSVYVKTTMGPSVEVLPSGTGQDTQIVVFATGETAAEDAADEVLGPDELEDFGDDTDAAKDL
>rl1_metja
VKEARELAKPRNFTQSFEFIATLKEIDMRKPENRIKTEVVAKAHDFFIAQADLMPLIGRYMGVILGPRGK
MPKPVPANANIKPLVERLKKTVVINTRDKPYFQVLVGNEKMTDEQIVDNIEAVLNVVAKKYEKGLLPHGR
GKEAKIAVIGTGDLAKQAEELGLTVIRKEEIEELGKNKRKLRKIYHIKDAYVKLTMGPAVKVK
>rl1_sulso
AIQNEWFLINQESMALAGRILGPALGPRGKFPTPLPNTADISEYINRFDKESLIEALKLALSTEYNVKRN
FTQSVEIILTFKGIDMKKGDLKLREIVPKRSVIVKTKDQPQVQVFIGTEDMKPEDLAENAIAVLNAIENK
AKVELPKQPSKAKRVLVVPSSEQLEYAKKASPKVVITREELQKLQGQKRPVKKLTNLRNIYVKTTMGKAV
KVK
