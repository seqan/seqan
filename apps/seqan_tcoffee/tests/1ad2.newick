(rk1_cyapa,(rl1_halvo,(rl1_metja,rl1_sulso)))
