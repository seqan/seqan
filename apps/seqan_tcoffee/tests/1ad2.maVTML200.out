>rk1_cyapa
EEQLYEPLEAI-QLLKQISTAKFIETVEVHCRLGIDPKYN--------------------
----------------------------DQQLRATVT-----------------------
-----------------------------KAGKLEFRNDRTGIVHVLFGKANFSEEALLG
NLKAVQEAVDRNRPSGVKGKYWKSIYITSTMSPSIEVDLPNGTGQTLRVAVITRGEKINE
ASTAG-ADLVGAEELIDEIQNGRLDFDCLIATPDMMPQVAKLGRVLGPRGLMPSPKGGTV
TFDLAQAINDF
>rl1_halvo
-------VDAVSRALDEAPGRNFRETVDLAVNLR-DLDLNDPSKRVDESIVADETDFFVA
EAGLMQDIGRYLGTVLGPRGKMPTPLQPDDDVVETVNRM---------------------
-----------------------------KNTVQLRSRDRRT-FHTRVGADDMTPDEIAE
NIDVIVRRLEATLEKGPL--NIDSVYVKTTMGPSVEV-LPSGTGQDTQIVVFATGETA--
AEDAA-DEVLGPDE-LEDFGDDT----------------DAAKDL---------------
-----------
>rl1_metja
----------VKEARELAKPRNFTQSFEFIATLK-EIDMRKPENRIKTEVVAKAHDFFIA
QADLMPLIGRYMGVILGPRGKMPKPVPANANIKPLVERL---------------------
-----------------------------KKTVVINTRDKPY-FQVLVGNEKMTDEQIVD
NIEAVLNVVAKKYEKG--------------------L-LPHGRGKEAKIAVIGTGDLAKQ
AEELG-LTVIRKEE-IEELGKNK----------------RKLRKIY-------HIKDAYV
KLTMGPAVKVK
>rl1_sulso
---------------------------------------------------AIQNEWFLI
NQESMALAGRILGPALGPRGKFPTPLPNTADISEYINRFDKESLIEALKLALSTEYNVKR
NFTQSVEIILTFKGIDMKKGDLKLREIVPKRSVIVKTKDQPQ-VQVFIGTEDMKPEDLAE
NAIAVLNAIENK-AKV--------------------E-LPKQPSKAKRVLVVPSSEQLEY
AKKASPKVVITREE-LQKLQGQK----------------RPVKKLT-------NLRNIYV
KTTMGKAVKVK
